doc/developers/advanced_installation.rst

```diff
@@ -20,129 +32,196 @@ basis to help users test bleeding edge features or bug fixes::
 .. _install_bleeding_edge:
 
 Building from source
-=====================
+====================
+
+.. _git_repo:
+
+#. Use `Git <https://git-scm.com/>`_ to check out the latest source from the
+   `scikit-learn repository <https://github.com/scikit-learn/scikit-learn>`_ on
+   `Github <https://github.com>`::
```

Comment:
There is probably no need to link to github.com? they will get there when clicking on the scikit-learn repo, and too many links can be confusing...

Reply:
Alright.